### DISCUSSION TITLE
[ALREADY A FEATURE] A way to loop individual songs.

### DISCUSSION THREAD

**Discussion by Teloshav** [navidrome/navidrome#2513] [Ideas]
THIS IS ALREADY A FEATURE 🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦🤦 

<s>The idea is that you're listening to a song from a playlist, album, or wherever and you want to loop the one song you're currently listening to from the Navidrome web interface.

I don't quite understand how this feature doesn't already exist on the web interface but I believe such a feature would be greatly appreciated by users.

Here's a screenshot from the desktop Spotify app on what this would appear as on the user interface:
![Spotify loop music UI](https://github.com/navidrome/navidrome/assets/131397191/5ac44cac-463c-4ab8-b6b9-af88b8b55b1b)</s>

- Teloshav

**Comment by metalheim** [comment]
You can though? In the player, there is an icon to toggle between
- straight play
- shuffle
- repeat all
- repeat 1 song

**Reply by Teloshav** [reply]
🤦 my  apologies...
